articles/scp-3211/src/en/anomalies/cat.ts

```diff
@@ -0,0 +1,35 @@
+import { Anomaly } from "../../anomaly"
+
+export default new Anomaly(false, {
+  imageUrl: "cat.jpg",
+  imageCaption: "SCP-3211 photographed in containment.",
+  objectClass: "Safe",
+  physicalDescription: `
+    SCP-3211 is a male Bengal cat that is estimated to be about
+    3 years old. It appears with a color and metal tag around its neck.
+
+    Unless being continually observed, the cat may teleport to another
+    position within its containment chamber. The animal does not apparently
```

Comment:
```suggestion
    position within its containment chamber. It does not apparently
```

Is this a universal truth - does it never notice the change? Current wording seems to refer to a single event